src/components/shared/SendModalBase/components/RecipientStep.tsx

```
@@ -34,16 +34,16 @@ export const RecipientStep: FC<RecipientStepProps> = ({
 }) => {
   return (
     <div
-      className="flex flex-col gap-4"
+      className="flex flex-col gap-4 min-h-0 flex-1"
       data-testid={testIdPrefix ? `${testIdPrefix}-recipient-step` : undefined}
     >
-      <Text size="sm" weight="medium" color="text-text-700">
+      <Text size="sm" weight="medium" color="text-text-700" className="flex-shrink-0">
```

ESLint

[error] Replace `·size="sm"·weight="medium"·color="text-text-700"·className="flex-shrink-0"` with `⏎········size="sm"⏎········weight="medium"⏎········color="text-text-700"⏎········className="flex-shrink-0"⏎······`

(prettier/prettier)